tests/foreman/ui/test_setting.py

```diff
@@ -952,3 +949,143 @@ def test_positive_update_token_duration_param(self):
+        """Updating SMTP params on Email tab fail
+
+        :id: d2a40c36-4a7e-45cb-a1b6-750ed49f222b
+
+        :steps:
+            1. Navigate to Administer > Settings > Email tab
+            2. Update delivery method select interface to SMTP
+            3. Update SMTP params with invalid configuration:
+                3.1. SMTP address
+                3.2. SMTP authentication
+                3.3. SMTP HELO/EHLO domain
+                3.4. SMTP password
+                3.5. SMTP port
+                3.6. SMTP username
+            4. Click "Test Email" button
+            5. Check error msg "Unable to send email, check server log for more
+                information" is shown
+            6. Check /var/log/foreman/production.log has error msg related
+                to email
+
+        :expectedresults: Email is not sent through SMTP
+
+        :CaseImportance: Critical
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_update_email_delivery_method_sendmail(self):
+        """Updating Sendmail params on Email tab
+
+        :id: 1651e820-7fc2-4295-acef-68737da8e1e2
+
+        :steps:
+            1. Navigate to Administer > Settings > Email tab
+            2. Update delivery method select interface to Sendmail
+            3. Sendmail params configuration:
+                3.1. Sendmail arguments
+                3.2. Sendmail location
+                3.3. Send welcome email
+            4. Update "Email reply address" and "Email subject prefix"
+            5. Click "Test Email" button
+            6. Check success msg "Email was sent successfully" is shown
+            7. Check sent email has updated values on sender and subject
+                accordingly
+
+        :expectedresults: Email is sent through Sendmail
+
+        :CaseImportance: Critical
+        """
+
+    @stubbed
+    @tier1
+    def test_negative_update_email_delivery_method_sendmail(self):
+        """Updating Sendmail params on Email tab fail
+
+        :id: 72391f12-68dd-4fce-b289-d3876564ce8a
+
+        :steps:
+            1. Navigate to Administer > Settings > Email tab
+            2. Update delivery method select interface to Sendmail
+            3. update Sendmail params with invalid configuration:
+                3.1. Sendmail arguments
+                3.2. Sendmail location
+                3.3. Send welcome email
+            4. Click "Test Email" button
+            5. Check error msg "Unable to send email, check server log for more
+                information" is shown
+            6. Check /var/log/foreman/production.log has error msg related
+                to email
+
+        :expectedresults: Email is not sent through Sendmail
+
+        :CaseImportance: Critical
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_email_yaml_config_precedence(self):
+        """Check configuration file /etc/foreman/email.yaml takes precedence
+        over UI. This behaviour will be default until Foreman 1.16.
```

Comment:
This test will fail for 1.16, Also we dont have decorator that skips test for perticular foreman/satellite version.